### PR TITLE
Prepare release 2.1.0

### PR DESCRIPTION
**Pull request by algesten**

### CODE REVIEW

**Comment by algesten** [comment]
@jsha #356 is referenced in the changelog since I believe we can merge it before releasing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 2.0.3
+ * Bugfix: allow status lines without a reason phrase (#316)
+ * Example: "cureq" to easier make ad-hoc command line tests (#330)
```
Comment (jsha) [inline CHANGELOG.md:3]:
```suggestion
 * Example: "cureq" to make ad-hoc command line tests easier (#330)
```

**Review by jsha — COMMENTED** [review]
This reminds me: what do you think of #339? Should the per-request method be able to set a longer timeout than the agent's timeout? We should answer this before cutting the new release, because it would be an API break to change it later.

**Review by jsha — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ureq"
-version = "2.0.2"
+version = "2.0.3"
```
Comment (jsha) [inline Cargo.toml:3]: Oh, one thing: this should actually take us to `2.1.0`. https://semver.org/

> Given a version number MAJOR.MINOR.PATCH, increment the:
>    MAJOR version when you make incompatible API changes,
>    MINOR version when you add functionality in a backwards compatible manner, and
>    PATCH version when you make backwards compatible bug fixes.

Comment (algesten) [reply Cargo.toml:3]: good catch!

File: CHANGELOG.md (hunk 1 above)
Comment (jsha) [reply CHANGELOG.md:3]: I think this tweak got lost in the rebase; otherwise everything looks good.

**Pull request merged**